app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -18,6 +18,8 @@ def self.create_stack(template, options = {})
     template_ref = template.new_record? ? nil : template
     new(:name                  => template.name,
         :ext_management_system => template.manager,
+        :verbosity             => template.variables["verbosity"].to_i,
```

Comment:
Do you want this to default to 0 if `verbosity` isn't in the variables?

Reply:
Yes I think that's acceptable